src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -2,15 +2,19 @@
 
 public class FindNumbers {
 
-    /**
-     * <p>Реализуйте метод getAllNumbers(String text) который принимает строку, cодержащую буквы,
-     * целые числа и иные символы.</p>
-     * <p>Требуется: все числа, которые встречаются в строке, поместить в отдельный целочисленный
-     * массив, каждый элемент этого массива умножить на 2. Метод должен возвращать этот массив.</p>
-     * <p>Пример: если дана строка "data 48 call 9 read13 blank0a", то в массиве должны оказаться
-     * числа 92, 18, 26 и 0.</p>
-     */
     public int[] getAllNumbers(String text) {
-        return null;
+        String str = text.replaceAll("[^-?\\d+$]", " ").trim();
+        String[] resStrings = str.split("\\s+");
+        int[] result = new int[resStrings.length];
+
+        for (int i = 0; i < resStrings.length; i++) {
+            result[i] = Integer.parseInt(resStrings[i]);
+        }
+
+        for (int i = 0; i < result.length; i++) {
+            result[i] *= 2;
```

Comment:
two loops are bad for performance, you can solve this task using one loop, optimize it